Example/panAndScroll/index.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import {
+  Animated,
+  Dimensions,
+  Platform,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+import {
+  PanGestureHandler,
+  TapGestureHandler,
+  ScrollView,
+  State,
+} from 'react-native-gesture-handler';
+
+import { USE_NATIVE_DRIVER } from '../config';
+import { LoremIpsum } from '../common';
+
+const windowWidth = Dimensions.get('window').width;
+const circleRadius = 30;
+
+export class TapOrPan extends Component {
+  constructor(props) {
+    super(props);
+    this._translateX = new Animated.Value(windowWidth / 2 - circleRadius);
+    this._lastX = 0;
+    this._onGestureEvent =
+      Platform.OS === 'ios'
```

Comment:
Is this really necessary? I prefer for all the examples to use animated and preferably with native driver which is one of the goals of the project to make that possible.

Reply:
Unsure if this would be better in a separate PR but mapping translationX on Android with a PanGestureHandler doesn't work as expected, it always seems to be a 10-20 pixels behind the touch point.

Reply:
I think I still would prefer to have the same code running on ios and android that would serve as a test and perhaps can be fixed by someone in the future.

Can you please change this so that we don't have a `Platform` check in this example and maybe add a short comment note about what needs to be fixed on android?

Reply:
Sounds good to me, this saves me tine I was about to make a gif to illustrate. I think your approach is the way to go that way the issue is at least apparent to people using the demo.